### PR TITLE
add warnings for non-existing values (fixes #136)

### LINKED ISSUE #119: Warn if someone is trying to connect to an element that doesn't have a view model
`<input {(value)}="something"/>`

instead of 

`<input {($value)}="something"/>`

should warn.

### LINKED ISSUE #136: Better warning when a binding can't set a value`
From http://forums.donejs.com/t/data-binding-multi-select-with-data-transformed/554/2

We should let people know when they are trying to set a value on something that doesn't exist.

### PR DESCRIPTION
**Pull request by christopherjbaker**
This may break tests in other repos if they have warning checks. (as my previous add-warnings PR did).

fixes #119
fixes #136

### CODE REVIEW

File: can-stache-bindings.js [hunk 1]
```diff
@@ -830,6 +830,22 @@ var canLog = require('can-util/js/log/log');
...
+		//!steal-remove-start
+		if (parentCompute && parentCompute.isComputed) {
+			if (typeof parentCompute() === "undefined") {
+				dev.warn('The parent (' + bindingInfo.parentName + ') you bound to child' +
+					' (' + bindingInfo.childName + ') is not defined in the scope!');
+			}
+		}
+
+		if (childCompute && childCompute.isComputed) {
+			if (typeof childCompute() === "undefined") {
```
Comment (justinbmeyer) [inline can-stache-bindings.js]: childCompute() === undefined

**Review by justinbmeyer — CHANGES_REQUESTED** [review]
I think for https://github.com/canjs/can-stache-bindings/issues/119, this needs to just see there's no view model on the element and warn if someone's binding to one that isn't there.  I also think we should only warn if we've got a scope->child and `scope` is undefined.

**Comment by christopherjbaker** [comment]
I'm not sure what to check for. Might need to pair on this to figure it out.

**Review by justinbmeyer — DISMISSED** [review]
I think this should look for `domData.get(el,"viewModel")`.

**Review by phillipskevin — APPROVED** [review]

**Pull request merged**